README.md

```diff
@@ -50,3 +56,9 @@
 
 ## Advance ML-on-Code awareness
 
+### App team activities
+
+- [ ] Speak at at least 1 conference
```

Comment:
The same this is a DevRel tasks, if they want someone on your team doing it, they are going to ask.

Reply:
Got it. 

@campoy I thought this might help to declare the intentions and set the right expectations